docs/troubleshooting/troubleshooting.md

```diff
@@ -38,4 +38,13 @@ The cause of the problem was that the firewall did not close the already existin
 tcpkill -9  -i ens192 src host ${KARMADA_APISERVER_IP} and dst port ${MEMBER_CLUTER_APISERVER_IP}
 ```
 
+## x509: certificate signed by unknown authority issue when using `karmadactl init`
 
+When useing the `karmadactl init` command to install Karmada, `karmada-apiserver` component raises the error log as follows:
+```log
+deploy.go:55] Post "https://192.168.24.211:32443/api/v1/namespaces": x509: certificate signed by unknown authority (possibly because of "crypto/rsa: verification error" while trying to verify candidate authority certificate "karmada")
+```
+
+Cause: Karmada has been installed on the cluster before. `karmada-etcd` uses the `hostpath` mode to mount the local storage. When `karmada-etcd` is uninstalled, residual data exists. We need to delete files in the `/var/lib/karmada-etcd` directory.
```

Comment:
```suggestion
Cause: Karmada has been installed on the cluster before. `karmada-etcd` uses the `hostpath` mode to mount the local storage. When `karmada-etcd` is uninstalled, residual data exists. We need to delete files in the  defualt directory`/var/lib/karmada-etcd`. If the karmdactl  [--etcd-data](https://github.com/karmada-io/karmada/blob/master/pkg/karmadactl/cmdinit/cmdinit.go#L119) parameter is used,please delete the corresponding directory.
```

Reply:
Thanks for your review, and I update it.